frontend/src/utils/formatters.ts

```diff
@@ -49,6 +62,46 @@ export function formatServiceArea(serializedServiceArea: string) {
   let { county, zip } = parseSerializedServiceArea(serializedServiceArea)
   return `${capitalizeWords(county)} / ${zip}`
 }
+/**
+ * Converts long string into array of shorter strings, with given maximum width.
+ */
+export function formatLabel(str: string, maxwidth: number) {
+  // TODO: Use lodash to shorten this function.
+  let sections: string[] = []
+  let words = str.split(' ')
+  let temp = ''
+
+  words.forEach(function (item, index) {
+    if (temp.length > 0) {
+      let concat = temp + ' ' + item
+
+      if (concat.length > maxwidth) {
+        sections.push(temp)
+        temp = ''
+      } else {
+        if (index === (words.length - 1)) {
+          sections.push(concat)
+          return
+        } else {
+          temp = concat
+          return
```

Comment:
What is that doing?

Reply:
it breaks a string into multiple lines based on a maxWidth of characters. I copypasta'd it from stack overflow 😳